README.md

```diff
@@ -60,8 +60,9 @@ $ docker pull ghcr.io/wouterdebie/locast2tuner:latest
 $ docker run -p 6077:6077 -v $HOME/.locast2tuner/:/app/config --name locast2tuner -d ghcr.io/wouterdebie/locast2tuner:latest
 ```
 
-If you'd like to use `docker-compose` you can use the sample [docker-compose.yml](https://github.com/wouterdebie/locast2tuner/blob/main/assets/docker/docker-compose.yml).
+If you'd like to use `docker-compose` you can use the sample [docker-compose.yml](https://github.com/wouterdebie/locast2tuner/blob/main/assets/docker/docker-compose.yml). 
```

Comment:
Why the trailing space?

Reply:
Two reasons:

1) It acts as a `<br>` in Markdown and tells it to enforce the next blank line.  But also...

2) Force of habit because I'm old and learned to type on typewriters where we used two spaces following a period. :-) 

In the above case, I may have tried to change wording but then retyped the original because I wasn't happy with the change.

Reply:
Gotcha.. The thing is that my editor automatically removes trailing spaces (which is a bad practice in programming), so next time I edit the README, the space will disappear :)

Reply:
Oh, haha - I tend to give the entire guide a once-over when I make an edit and fix little things here and there, so I'm sure I've fixed that without noticing.

Not a big deal.  Most people don't notice.